types/react-tagsinput/index.d.ts

```diff
@@ -9,18 +9,42 @@ import * as React from "react";
 export as namespace ReactTagsInput;
 export = TagsInput;
 
+type Tag = any;
```

Comment:
Can it really be anything? Or is it `string | object` or just `string` in practice? Can you add a test with a real example?